conans/test/unittests/client/conanfile_loader_test.py

```diff
@@ -117,8 +117,7 @@ def test_load_options_error(self):
         save(file_path, conanfile_txt)
         loader = ConanFileLoader(None, None)
         with self.assertRaisesRegex(ConanException,
-                                   r"Error while parsing \[options\] in conanfile.txt\n"
-                                   "Options should be specified as 'pkg/\*:option=value'"):
+                                   r"Error while parsing"):
```

Comment:
This change is showing some negative impact in the UX, users will not longer know that their error is from ``conanfile.txt`` declared ``[options]``